openadapt/config.py

```diff
@@ -100,8 +100,26 @@
     list(stop_str) for stop_str in STOP_STRS
 ] + SPECIAL_CHAR_STOP_SEQUENCES
 
+ENV_FILE_PATH = ".env"
+ENV_EXAMPLE_PATH = ".env.example"
```

Comment:
Please rename to `ENV_EXAMPLE_FILE_PATH`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/223